### DISCUSSION TITLE
can't catch liquidations

### DISCUSSION THREAD

**Discussion by masatoshi111** [bmoscon/cryptofeed#646] [Q&A]
Hi team,
I can't catch any callbacks to LIQUIDATIONS of BinanceFutures,
Could you help me please?

### Code
```
from cryptofeed import FeedHandler
from cryptofeed.exchanges import Binance, BinanceFutures
from cryptofeed.defines import LIQUIDATIONS

async def liquidations(liquidation, receipt_timestamp):
    print(f"Liquidation received at {receipt_timestamp}: {liquidation}")

def main()
    config = {'log': {'filename': 'demo.log', 'level': 'DEBUG'}}
    f = FeedHandler(config=config)
    pairs = BinanceFutures.symbols()[:30]
    f.add_feed(BinanceFutures(symbols=pairs, channels=[LIQUIDATIONS], callbacks={LIQUIDATIONS: liquidations}))
    f.run()

if __name__ == '__main__':
    main()
```

### Debug log
```
2021-09-21 14:05:16,134 : INFO : Config: use dict containing the following main keys: log
2021-09-21 14:05:16,136 : DEBUG : BINANCE_FUTURES: reading symbol information from https://fapi.binance.com/fapi/v1/exchangeInfo
2021-09-21 14:05:16,137 : DEBUG : HTTPSync: requesting data from https://fapi.binance.com/fapi/v1/exchangeInfo
2021-09-21 14:05:16,371 : DEBUG : BINANCE_FUTURES.ws.1: connecting to wss://fstream.binance.com/stream?streams=~~~
```

**Comment by agijsberts** [comment]
Your code runs fine here (except for a missing colon after `def main()`).

Did you wait long enough? By nature, liquidations only happen with market movements, so when markets are calm it can happen that there won't be any for quite some time.

**Reply by masatoshi111** [reply]
thanks for replying
I have waited long enough to happen liquidations (Bybt showed some liquidations at same time)
I can get open interest with similar code....

i installed just with `pip inatall cryptofeed[all]`, does it need setting to use after install?

**Reply by agijsberts** [reply]
No, that should be enough.

Can you try to downgrade `websockets` and try again?
```bash
pip install websockets==9.1
```
Alternatively, you can also try the latest git version of `cryptofeed`.

**Reply by masatoshi111** [reply]
Fixed with `pip install websockets==9.1` !
thanks

**Reply by agijsberts** [reply]
Great, you hit a bug with websocket compatibility with certain exchanges. The latest git cryptofeed contains a workaround, so with the next 2.0.1 release it will no longer be needed to downgrade `websocket`.
